### PR TITLE
Firstname updates

### PR DESCRIPTION
**Pull request by CourtneyLO**
## Feature Description
This PR changes the placeholder for the first name form field and adds partial documentation.

## Link to Ticket / Card:
[Trello Ticket](https://trello.com/c/Yg0RZ6sp/1238-1-change-firstname-form-field-placeholder)

## Screenshots:

New:
image

Old:
image

## Has the necessary documentation been created / updated?
- [x] Yes
- [ ] Not required for this ticket

## Has this been given a review by Design/UX?
- [ ] Yes
- [ ] Not required for this ticket

### CODE REVIEW

File: docs/PARTIALS.md [hunk 1]
```diff
@@ -31,6 +32,16 @@ The form fields to be displayed in this fieldset.
...
 {{/ n-conversion-forms/partials/fieldset }}
 ```
+#### firstname
+
+A form field for adding a user's first name.
+
+##### Options
+
++ `value`: string - Text to pre-populate the `firstname` field.
++ `hasError`: boolean - true - adds `o-forms--error` class
```
Comment (liamkeaton) [inline docs/PARTIALS.md]: Missing `isDisabled` option

File: docs/PARTIALS.md [hunk 2]
```diff
@@ -31,6 +32,16 @@ The form fields to be displayed in this fieldset.
...
++ `isDisabled`: boolean - true - disables the form field
+{{> n-conversion-forms/partials/firstname value=John }}
```
Comment (liamkeaton) [inline docs/PARTIALS.md]:
```suggestion
```
{{> n-conversion-forms/partials/firstname value=John }}
```
```

Comment (liamkeaton) [inline docs/PARTIALS.md]: Should this have code comments around it?

Comment (CourtneyLO) [reply docs/PARTIALS.md]: Do you mean:
Do you mean
```
{{> n-conversion-forms/partials/firstname value=John }}
```

Comment (liamkeaton) [reply docs/PARTIALS.md]: Yup, think you can use \`\`\`handlebars

**Pull request merged**